frontend/packages/kubevirt-plugin/src/components/modals/menu-actions-modals/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { history } from '@console/internal/components/utils';
+import { getName, getNamespace } from '@console/shared/src/selectors/common';
+import { VMILikeEntityKind } from 'packages/kubevirt-plugin/src/types/vmLike';
+
+export const redirectAfterVMDeletion = (vmi: VMILikeEntityKind) => {
```

Comment:
can we use this one for the template as well? And parametrize the `/virtualization` part? I guess better name could be redirectToList

Reply:
yes, made it one method, much nicer now